packages/logging/src/logging-and-telemetry.ts

```diff
@@ -284,10 +395,8 @@ class LoggingAndTelemetry implements MongoshLoggingAndTelemetry {
         }
         this.busEventState.telemetryAnonymousId =
           newTelemetryUserIdentity.anonymousId;
-        this.analytics.identify({
-          anonymousId: newTelemetryUserIdentity.anonymousId,
-          traits: getUserTraits(),
-        });
+
+        void identify();
```

Comment:
Total nit, but there's no promise you're not awaiting, so the `void` pattern is probably not necessary?